nengo_viz/swi.py

```diff
@@ -49,6 +49,7 @@ def ws_data(self, client):
 import select
 import string
 import sys
+import errno
```

Comment:
They weren't alphabetized in the first place, so I didn't bother ... though I agree.